### PR TITLE
tmuxinator: 1.1.4 -> 2.0.0

### PR DESCRIPTION
**Pull request by cust0dian-old**
###### Motivation for this change
https://github.com/tmuxinator/tmuxinator/releases/tag/v1.1.5
https://github.com/tmuxinator/tmuxinator/releases/tag/v2.0.0

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by marsam** [comment]
2.0.0 was released a few hours ago https://github.com/tmuxinator/tmuxinator/releases/tag/v2.0.0
Would you mind updating it?

**Comment by cust0dian** [comment]
@marsam sure, done!

**Comment by doronbehar** [comment]
@cust0dian can you make sure the completion files are installed as well?

https://github.com/tmuxinator/tmuxinator/blob/master/README.md#completion

**Comment by cust0dian** [comment]
@doronbehar sure, took a stab at it, although would greatly appreciate if someone more experienced with Ruby packaging double-checked my approach.

File: pkgs/tools/misc/tmuxinator/default.nix [hunk 1]
```diff
@@ -37,6 +37,12 @@ buildRubyGem rec {
 
   propagatedBuildInputs = [ erubis thor xdg ];
 
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion $GEM_HOME/gems/${gemName}-${version}/completion/tmuxinator.{bash,zsh,fish}
```
Comment (cust0dian) [inline pkgs/tools/misc/tmuxinator/default.nix:43]: Not 100% sure getting to gem source via `$GEM_HOME/gems/${gemName}-${version}/` is _the_ way to do it (decided to use this approach as I've seen it in other places)

**Comment by cust0dian** [comment]
@GrahamcOfBorg eval

**Comment by Mic92** [comment]
Result of `nixpkgs-review pr 86334` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
<br>- tmuxinator
</details>

**Pull request merged**